manager.go

```diff
@@ -149,24 +148,24 @@ type Manager struct {
 
 	// DumpHandler - Callback function called when manager.DumpMaps() is called
 	// and dump the current state (human-readable)
-	DumpHandler func(manager *Manager, mapName string, currentMap *ebpf.Map) string
+	DumpHandler func(manager *Manager, mapName string, currentMap *ebpf.Map, w io.Writer)
```

Comment:
since all the handlers are going to need to change anyways, could you put the writer first? It is mostly a style thing, but since it the "always used" argument, I tend to put those earlier.